dist/api_data.py

```diff
@@ -454,9 +454,6 @@ def __ge__(self, other):
 ]
 
 index_only_config = [
-    Config('extractor', 'none', r'''
```

Comment:
Is this config internal only?

Reply:
No, it is not internal only. See https://source.wiredtiger.com/develop/custom_extractors.html#custom_extractors_intro Version 11.3.0 WiredTiger > Reference Guide > Writing WiredTiger applications > Custom Extractors > Introduction to Custom Extractors.